internal/build/src/tasks/types-definitions.ts

```diff
@@ -1,158 +0,0 @@
-import process from 'process'
-import path from 'path'
-import { mkdir, readFile, writeFile } from 'fs/promises'
-import consola from 'consola'
-import * as vueCompiler from 'vue/compiler-sfc'
-import glob from 'fast-glob'
-import chalk from 'chalk'
-import { Project } from 'ts-morph'
```

Comment:
It seems the `ts-morph` is no longer needed, perhaps we should remove it.

Reply:
Removed

Reply:
It seems that the `ts-morph` has not been removed from the package.json

Reply:
Oops😅, I accidentally didn't commit the files. Will push them in the next commit.